### PR TITLE
add help menu for cmd 'mc admin user add'

### LINKED ISSUE #4326: mc admin user add does not display help menu
## Expected behavior
mc admin user add should display help
## Actual behavior
image

## Steps to reproduce the behavior
mc admin user add

## mc --version
mc version RELEASE.2022-10-22T03-39-29Z

### PR DESCRIPTION
**Pull request by dormanze**
## Description
add help menu for cmd 'mc admin user add'

## Motivation and Context
fix #4326

## How to test this PR?
mc admin user add
image

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fmc%2fpull%2fNNNNN)

### CODE REVIEW

File: cmd/admin-user-add.go [hunk 1]
```diff
@@ -75,8 +75,8 @@ EXAMPLES:
 func checkAdminUserAddSyntax(ctx *cli.Context) {
 	argsNr := len(ctx.Args())
 	if argsNr > 3 || argsNr < 1 {
-		fatalIf(errInvalidArgument().Trace(ctx.Args().Tail()...),
-			"Incorrect number of arguments for user add command.")
+		// fix #4326 , display help menu.
```
Comment (harshavardhana) [inline cmd/admin-user-add.go:78]:
```suggestion
```

File: cmd/admin-user-add.go [hunk 2]
```diff
@@ -75,8 +75,8 @@ EXAMPLES:
...
+		showCommandHelpAndExit(ctx, "add", 1)
```
Comment (harshavardhana) [inline cmd/admin-user-add.go:79]:
```suggestion
		showCommandHelpAndExit(ctx, ctx.Command.Name, 1)
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: cmd/admin-user-add.go (hunk 1 above)
Comment (dormanze) [reply cmd/admin-user-add.go]: Thanks for your reply, I will delete this line.

File: cmd/admin-user-add.go (hunk 2 above)
Comment (dormanze) [reply cmd/admin-user-add.go]: I found the same problem in other help information, and I will fix it together.

File: cmd/admin-bucket-remote-bandwidth.go [hunk 3]
```diff
@@ -140,10 +140,10 @@ func checkAdminBwInfoSyntax(ctx *cli.Context) {
 		u != "Bi" &&
 		u != "B" &&
 		u != "" {
-		showCommandHelpAndExit(ctx, "bandwidth", globalErrorExitStatus)
```
Comment (harshavardhana) [inline cmd/admin-bucket-remote-bandwidth.go:143]: you don't need to do that I have a more comprehensive change #4338

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: cmd/admin-bucket-remote-bandwidth.go (hunk 3 above)
Comment (dormanze) [reply cmd/admin-bucket-remote-bandwidth.go]: ok,copy.

**Review by harshavardhana — APPROVED** [review]

**Review by klauspost — APPROVED** [review]
LGTM

**Pull request merged**